Add fallback value for enums (#33)
Fortnite-API/py-wrapper: docs/changelog.rst

~~~~~~~~~~~~~~~~
- ``CreatorCode.status`` and ``CreatorCode.disabled`` have been removed, since they where never returned by this endpoint. Disabled creator codes always raise :class:`fortnite_api.NotFound` when trying to fetch them.
- ``CreatorCode.verified`` has been removed, since it isn't used within the affiliate system. It always returns ``False``.

> - All enums now use an internal "Enum-like" class to handle unknown values, instead of the built-in ``enum.Enum``. This potentially breaks type checks, but it should not break core functionality.

Reviewer: ```suggestion
- All enums now use an internal "Enum-like" class to handle unknown values, instead of the built-in :class:`py.enum.Enum`. This potentially breaks type checks, but does not break core functionality or change the enum interface; you can use them the same.
```

New Features
~~~~~~~~~~~~
- Added support for :attr:`fortnite_api.CosmeticType.SHOES`.

Miscellaneous
~~~~~~~~~~~~~
- Add safeguards against Epic Games' API changing or providing invalid values in API responses.

>     - All enums now can handle unknown values via an internally defined "Enum-like" class. This means that if the API returns a value that is not in the enum, it will be stored as an attribute on the enum object.

Reviewer: ```suggestion
    - All enums now can handle unknown values via an internally defined "Enum-like" class. If the API returns a value not in the enum, it will be stored as an attribute on the enum object. The interface for using this class is the same as using :class:`py.enum.Enum`.
```

~~~~~~~~~~~~
- Added support for :attr:`fortnite_api.CosmeticType.SHOES`.

Miscellaneous
~~~~~~~~~~~~~

> - Add safeguards against Fortnite changes or messed up values within the API.

Reviewer: Could be made more clear here
```suggestion
- Add safeguards against Epic Games' API changing or providing invalid values in API responses.
```

Reviewer: I'd also note that all "Enum-like" classes in the breaking changes section are not actually `enum.Enum`s anymore, in case one of our users is using an instance check.

Author: ~Shouldn't instance checks still work, since we just subclassing Enum? But yeah, if you expect a specific type it could break.~ Nvm. completely forgot that the Enum import was just for type checking. Gonna add that to the changelog.

Author: I updated the changelog. Feel free to suggest changes in wording.

>     - All enums now can handle unknown values. This means that if the API returns a value that is not in the enum, it will be stored as an attribute on the enum object.

Reviewer: ```suggestion
    - All enums now can handle unknown values via an internally defined "Enum-like" class. This means that if the API returns a value that is not in the enum, it will be stored as an attribute on the enum object.
```
